fe1-web/docs/README.md

```diff
@@ -631,7 +631,7 @@ If you write E2E tests, please be mindful of making them maintainable through th
 Please reach out to the DEDIS Engineering team members to deploy a build to an
 internet accessible host.
```

Comment:
actually, we could mention that a GitHub release now automatically deploys to fe1.personhood.online

Reply:
True 😁